department-of-veterans-affairs/caseflow
# Finish Refiling Intake | Create contentions in VBMS

Create contentions in VBMS when finishing a refiling intake

## Testing instructions

Finish an refiling intake. See that the issues have a contention_reference_id
from the fake contentions in VBMSService. Also look in the logs to see that VBMSService.create_contentions!
was called.

The function was tested in UAT, but the entireity of the feature will be
tested more thoroughly before we lift the feature flag.

connects #3966
